### ISSUE TITLE
DDragon runesReforged.json does not include the three stat slots

### ISSUE THREAD

**Issue by JanOuborny** [RiotGames/developer-relations#26]
I can't find the data for the three stat slots, which you can select in a rune page. According to my understanding, they should be included in the `runesReforged.json` file.

**Comment by Nevaehy** [comment]
Couldn't find any information about new rune slots also. Neither in runesReforged.json nor as separated image by id (5001, 5008 and others).

**Comment by RiotTuxedo** [comment]
Good catch, this was original made in an announcement prior to moving over to riotgames.com. Here's the information you're looking for (I think). I'll look for a more permanent place for this.

There are currently three rows, multiple options per row, and six options total.
```
Row 1: 5008, 5005, 5007
Row 2: 5008, 5002, 5003
Row 3: 5001, 5002, 5003
```

```
5001 Scaling Health (15-90 HP, lvls 1-18)
5002 Armor (5 Armor)
5003 Magic Resist (6 MR)
5005 Attack Speed (9% Attack Speed)
5007 Scaling Cooldown Reduction (1-10% CDR, lvls 1-18)
5008 Adaptive Force (6 AD or 10 AP)
```

**Comment by Nevaehy** [comment]
> Good catch, this was original made in an announcement prior to moving over to riotgames.com. Here's the information you're looking for (I think). I'll look for a more permanent place for this.

Thank you for the information! 
But I still have a question. Is it possible to pull this data from ddragon or somewhere else? I'm asking to know if it's possible to avoid hard coding :)
And I wonder where to find the images of these new options in case if this data cannot be pulled from ddragon.

**Comment by RiotTuxedo** [comment]
Eventually I would expect so, but not currently.

**Comment by stelar7** [comment]
> or somewhere
CDragon has the data/assets atm, so you can use that.
[Perks.json](https://raw.communitydragon.org/pbe/plugins/rcp-be-lol-game-data/global/default/v1/perks.json) [Images](https://raw.communitydragon.org/pbe/plugins/rcp-be-lol-game-data/global/default/v1/perk-images/)

**Comment by Nevaehy** [comment]
> 
> 
> > or somewhere
> 
> CDragon has the data/assets atm, so you can use that.
> [Perks.json](https://raw.communitydragon.org/pbe/plugins/rcp-be-lol-game-data/global/default/v1/perks.json) [Images](https://raw.communitydragon.org/pbe/plugins/rcp-be-lol-game-data/global/default/v1/perk-images/)
Thanks! It's gonna help.

**Comment by JoschuaGoetz** [comment]
@RiotTuxedo 
Is there anything new on this on Riot's side ? 
Using CDragon is an option but this should be included in DataDragon as well. I don't want to depend on two separate data sources when there should really be just a single one.

**Comment by JoschuaGoetz** [comment]
> Good catch, this was original made in an announcement prior to moving over to riotgames.com. Here's the information you're looking for (I think). I'll look for a more permanent place for this.
> 
> There are currently three rows, multiple options per row, and six options total.
> 
> ```
> Row 1: 5008, 5005, 5007
> Row 2: 5008, 5002, 5003
> Row 3: 5001, 5002, 5003
> ```
> 
> ```
> 5001 Scaling Health (15-90 HP, lvls 1-18)
> 5002 Armor (5 Armor)
> 5003 Magic Resist (6 MR)
> 5005 Attack Speed (9% Attack Speed)
> 5007 Scaling Cooldown Reduction (1-10% CDR, lvls 1-18)
> 5008 Adaptive Force (6 AD or 10 AP)
> ```

@RiotTuxedo with the new shards on PBE, can we get an updated list ?
